packages/runner/src/suite.ts

```diff
@@ -13,7 +13,7 @@ export const suite = createSuite()
 export const test = createTest(
   function (name: string | Function, fn?: TestFunction, options?: number | TestOptions) {
     if (getCurrentTest())
-      throw new Error('Nested tests are not allowed')
+      throw new Error('Calling the "test" or "it" function inside another "test" or "it" function is not allowed. Please, put it inside "describe" so it can be properly collected.')
```

Comment:
```suggestion
      throw new Error('Calling the "test" or "it" function inside another "test" or "it" function is not allowed. Please put it inside "describe" so it can be properly collected.')
```